File: Program/ActEditor/bin/pl/wroc/pwr/student/acteditor/test/Load.js
Clazz.declarePackage ("pl.wroc.pwr.student.acteditor.test");
Clazz.load (null, "pl.wroc.pwr.student.acteditor.test.Load", ["java.io.BufferedReader", "$.FileReader"], function () {
c$ = Clazz.declareType (pl.wroc.pwr.student.acteditor.test, "Load");
c$.main = Clazz.defineMethod (c$, "main", 
function (args) {
var br = null;
try {
var sCurrentLine;
br =  new java.io.BufferedReader ( new java.io.FileReader ("D:\\Opera Unite\\EDAP\\Program\\ActEditor\\resources\\asd.txt"));
while ((sCurrentLine = br.readLine ()) != null) {
System.out.println ("\"" + sCurrentLine.replaceAll ("\"", "\\\\\"") + "\\n\"+");
}
} catch (e) {
if (Clazz.instanceOf (e, java.io.IOException)) {
e.printStackTrace ();
} else {
throw e;
}
} finally {
try {
if (br != null) br.close ();
} catch (ex) {
if (Clazz.instanceOf (ex, java.io.IOException)) {
ex.printStackTrace ();
} else {
throw ex;
}
}
}
}, "~A");
});

File: Program/ActEditor/src/pl/wroc/pwr/student/acteditor/test/Load.java
package pl.wroc.pwr.student.acteditor.test;

import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;

public class Load {

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		BufferedReader br = null;

		try {
			String sCurrentLine;
			br = new BufferedReader(new FileReader("D:\\Opera Unite\\EDAP\\Program\\ActEditor\\resources\\asd.txt"));
			while ((sCurrentLine = br.readLine()) != null) {
				System.out.println("\"" + sCurrentLine.replaceAll("\"", "\\\\\"") + "\\n\"+");
			}
		} catch (IOException e) {
			e.printStackTrace();
		} finally {
			try {
				if (br != null)
					br.close();
			} catch (IOException ex) {
				ex.printStackTrace();
			}
		}
	}

}
